ignite/handlers/checkpoint.py

```diff
@@ -84,7 +84,8 @@ class Checkpoint(Serializable):
             :class:`~ignite.engine.engine.Engine` object, and returning a score (`float`). Objects with highest scores
             will be retained.
         score_name: If ``score_function`` not None, it is possible to store its value using
-            ``score_name``. See Notes for more details.
+            ``score_name``. If ``score_function`` is None, can be used alone to fetch ``score_function``.
+            See Notes for more details.
```

Comment:
Without sending user to Notes below, we can just say that if `score_function is None`, we are using `Checkpoint.get_default_score_fn(score_name)` as default.